torch/onnx/__init__.py

```diff
@@ -197,6 +197,7 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
             Since this optimization adjusts model initializers, it will be disabled if
             export_params = False or keep_initializers_as_inputs = True.
         example_outputs (T or a tuple of T, where T is Tensor or convertible to Tensor, default None):
+            [Deprecated and ignored. Will be removed in next PyTorch release, do this part in the internal function],
```

Comment:
I don't think we need to mention internal function. The internal function's argument will be unused as well.